public/app/plugins/datasource/elasticsearch/datasource.ts

```diff
@@ -34,6 +37,7 @@ const ELASTIC_META_FIELDS = [
   '_ignored',
   '_routing',
   '_meta',
+  'sort',
```

Comment:
Doah! I meant to remove that. I was trying to add the sort key to the results because we need that for the `search_after` value, I meant to clean that up.